sdk/src/feature_set.rs

```diff
@@ -732,6 +732,10 @@ pub mod enable_zk_transfer_with_fee {
     solana_sdk::declare_id!("zkNLP7EQALfC1TYeB3biDU7akDckj8iPkvh9y2Mt2K3");
 }
 
+pub mod additional_vote_stake_threshold {
```

Comment:
these changes shouldn't need a feature gate. the fd team and operators would i'm sure be happy if we didn't break consensus unnecessarily

Reply:
Removed the feature gate